hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/ChunkCreator.java

```diff
@@ -228,14 +236,15 @@ Chunk getChunk(CompactingMemStore.IndexType chunkIndexType, int size) {
    */
   Chunk getJumboChunk(int jumboSize) {
     int allocSize = jumboSize + SIZEOF_CHUNK_HEADER;
-    if (allocSize <= dataChunksPool.getChunkSize()) {
+
+    if (allocSize <= this.getChunkSize()) {
```

Comment:
Why change from dataChunksPool.getChunkSize to getChunkSize?

Reply:
Because the `ChunkCreator.dataChunksPool` maybe null(If we disable the chunk pool), which may throw `NullPointerException`. 
`getChunkSize` is actually `getChunkSize(ChunkType.DATA_CHUNK)`, which consider the case  `ChunkCreator.dataChunksPool` is null.

Reply:
Then let's call getChunkSize(ChunkType.DATA_CHUNK) directly, to avoid confusing here.

Reply:
OK,  modified folloeing your suggestion.